### PR TITLE
Liberation Station 13

### PR DESCRIPTION
**Pull request by ErrorsWindows**
I was going to make this yesterday's pull request but I fell into a food coma about two thirds of the way through because I ate 3 chicken sandwiches and half a pound of nachos in one sitting

## What this does
liberation

### A perfect addition to any nuke ops, cargo traitor, castle station round, or really whenever squidmoko is online

The store leading the fight for freedom is going galactic with NEW portable vendors giving you a taste of America wherever you are! Ammu-Station has ALL the equipment you need to protect your department from the **EVILS** of a LIBERAL society! Fixed, mounted, shoulder-held shotguns, submachine guns, and all manner of heat-seeking whatever, and JUST IN to celebrate the World Wars, TRACER BULLETS in red, green, blue, purple, yellow, brown, and grey so you can protect your co-workers in the dark maintenance tunnels!
End the week in style on Make My Day Saturdays with 2-for-1 on all landmines! Got Shoal War Syndrome? Get $10 off all machinegun rentals! Start the week at Ammu-Station on Militia Mondays to exercise YOUR Second Amendment rights and get 15% off all ammunition purchases! We're the only gun store that lets you try it before you buy it! No credit? No problem: No money down, 90 days, same as cash, shoot now pay later! 
Come by Ammu-Station THIS MEMORIAL DAY to register to win an anti-mech missile launcher _actually_ used when we whooped the GDR's ass and see the Ammu-Station film festival with FREE screenings of the documentary _Grey Dawn (2077)_!

- Over 100 various All-American food and drink items, including burgers, fries, beer, and cola, 4 dozen donuts for the spacemen and spacewomen upholding peace
- Overall enough burgers and beer to make a crew obese 3 times over
- That's not to forget the dozen guns and hundreds of rounds of ammunition

## Why it's good
if you don't like the gun store you don't like AMERICA

image

### But why for adminbus?
Somebody familiar with the windows lore will remember that month in like, October or something where I tried to flesh out emag cargo a bit more
It was a bit puerile and I don't like it anymore and I think the reason why it didn't really catch on was because I didn't market it very well, but a conversation I had a couple days ago reminded me to take a second chance at this
Now I've got a bit more hair on my groin and am a bit wiser about atomicity

I don't _want_ this to be adminbus forever so I do hope it gets implemented somehow

- Emag cargo in the same vein as the nazi and soviet vendor, the rather ridiculous amount of weapons inside will probably be too absurd for most [considering how cargo players already are whenever they hear a beeping noise come from the other side of the maintenance door](https://youtu.be/g-1OYN9HH9c)
- [The nuke ops uplink or some other kind of human syndicate item](https://youtu.be/tX5ZRE26YWM), the rather ridiculous amount of weapons inside will probably be too absurd for some
- [Mapped into castle station](https://youtu.be/l3ZEL_LgP7s), the rather ridiculous amount of weapons inside will probably fit right in
- Remain adminbus, which will be lame and unfun

## Changelog
:cl:
 * rscadd: AmmuStation vending machine, yea this one, you know the one :)
 * rscadd: American flag bedsheet, Evel Knievel jumpsuit, and eagle spacesuit

### CODE REVIEW

**Comment by ErrorsWindows** [comment]
A fun fact while I was making this:
[There's apparently already an ammo vendor sprite?](https://github.com/vgstation-coders/vgstation13/pull/1163)
Screenshot (2048)
Did you know that? I didn't know that, there aren't any vending machines coded to have that sprite

In fact there are a bunch of unused sprites like the nutrimat (not the nutrivend)
Screenshot (2049)

**Comment by gurfan** [comment]
>all the products are free

**Comment by Eneocho** [comment]
Make it have a [42% chance to give you the obese gene](https://www.forbes.com/health/body/obesity-statistics/) on use or no deal.

**Comment by Eneocho** [comment]
`>AmmoStation instead of Ammu-Nation`
Come on windows, you should know better.
image

**Comment by BomberBro** [comment]
Use the sprite already in the game, yours needs work.

**Comment by Inorien** [comment]
> > all the products are free

image

**Comment by juanototo2** [comment]
Map this into castle, along with an independent gunshop owner role; so that it reaches its darkrp final form. Barring that, just leave it in its own cubbyhole near where sec used to be.

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -126,3 +126,36 @@
...
+		allowed = list(
+		/obj/item/weapon/gun,
+		/obj/item/device/flashlight,
+		/obj/item/weapon/tank,
+		/obj/item/weapon/melee/baton,
+		/obj/item/weapon/reagent_containers/spray/pepper,
+		/obj/item/ammo_storage,
+		/obj/item/ammo_casing,
+		/obj/item/weapon/handcuffs,
+		/obj/item/weapon/bikehorn/baton,
```
Comment (pedr69) [inline code/modules/clothing/spacesuits/miscellaneous.dm:154]: >honk baton fits on the suit
>but not a bikehorn or a stunbaton

someone give me a brief rundown on windowserrors lore

**Comment by Kromkar** [comment]
>windows lore

Don't take yourself so seriously dude lol

**Comment by SECBATON-GRIFFON** [comment]
make tracer rounds explode fuel tanks or cars a bit better since in real life shooting fuel does nothing explosive but it actually does with tracer bullets

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 2]
```diff
@@ -126,3 +126,36 @@
...
+	icon_state = "griffinhat"
+	item_state = "griffinhat"
+	armor = list(melee = 40, bullet = 40, laser = 30, energy = 25, bomb = 90, bio = 100, rad = 20) //90 bomb armor so you can do the action movie thing of getting thrown away from the explosion by the shockwave, not sure how it will interact with fragmentation
+	species_restricted = list("exclude",VOX_SHAPED) //you're already a bird dumb fuck
+	siemens_coefficient = 0.7
+	pressure_resistance = 40 * ONE_ATMOSPHERE
+
+/obj/item/clothing/suit/space/freedom
+	name = "eagle suit"
+	desc = "An advanced, light suit, fabricated from a mixture of synthetic feathers and space-resistant material. A gun holster appears to be intergrated into the suit and the wings are stuck in 'freedom' mode."
```
Comment (SonixApache) [inline code/modules/clothing/spacesuits/miscellaneous.dm:142]: typo on purpose?

**Comment by ghost** [comment]
Ugly sprites + immersion breaking. Space Nazis and Soviets are a part of SS13 lore (not so sure about the former) but Murica along with China and all other specific Earth countries have either lost relevance outside of their planet or have merged into one planet-wide government as evident by Sol Common and the fact that Space Station 13 is owned not not by some government but by an intergalactical business,

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 3]
```diff
@@ -126,3 +126,36 @@
...
+		/obj/item/weapon/melee/baton,
+		/obj/item/weapon/reagent_containers/spray/pepper,
+		/obj/item/ammo_storage,
+		/obj/item/ammo_casing,
+		/obj/item/weapon/handcuffs,
+		/obj/item/weapon/baton,
+		/obj/item/weapon/blunderbuss,
+		/obj/item/weapon/legcuffs/bolas,
+	)
+	species_restricted = list("exclude",VOX_SHAPED)
```
Comment (Securitymain) [inline code/modules/clothing/spacesuits/miscellaneous.dm:158]: i know the codebase is not consistant on space after comma, but this is disgusting

**Comment by Blithering** [comment]
I felt oddly compelled to vote in favour of this this today.

**Comment by ErrorsWindows** [comment]
man fuck it i'm too lazy for this maintainer shit

**Comment by Eneocho** [comment]
bro you just need to fix a typo and that's that, why close this?

**Comment by ErrorsWindows** [comment]
yeabu WHERE

**Comment by Kromkar** [comment]
lol

**Pull request closed without merging**